### PR TITLE
function docs boilerplate

### PR DESCRIPTION
**Pull request by jlerche**

### CODE REVIEW

**Comment by cmeiklejohn** [comment]
We should start by getting some documentation in around the format needed to supply to join and leave.

**Comment by jlerche** [comment]
So basically that would  be the form of the map that `myself` returns and its keys and values, more or less?

**Comment by cmeiklejohn** [comment]
Thats right

**Comment by jlerche** [comment]
Added a blurb for `myself/0`. There's an error that gets spammed in the console

```
17:20:49.197 [error] Node #{channels => [undefined],listen_addrs => [#{ip => {127,0,0,1},port => 57449}],name => 'foo@127.0.0.1',parallelism => 1} is not yet connected during send!
```
Not really sure what it's for, but maybe a general config issue? We should have instructions on how to connect two nodes running on the same host as well as remotely.

edit: I tried poking around, wasn't sure if `partisan_peer_service_client` genserver got started, but maybe it's not relevant here?

File: lib/expartisan.ex [hunk 1]
```diff
@@ -91,6 +106,22 @@ defmodule ExPartisan do
...
+
+      iex(foo@127.0.0.1)> ExPartisan.send_message(bar, :c.pid(0,308,0), :hello)
+      :ok
+
+      iex(bar@127.0.0.1)4> flush
+      :hello
+      :ok
+
+
+  Note that `server_ref` can be something other than a pid.
```
Comment (cmeiklejohn) [inline lib/expartisan.ex:124]: ``server_ref`` could be specified here using the type signature from the Erlang documentation

File: mix.lock [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "makeup": {:hex, :makeup, "0.5.5", "9e08dfc45280c5684d771ad58159f718a7b5788596099bdfb0284597d368a882", [:mix], [{:nimble_parsec, "~> 0.4", [hex: :nimble_parsec, repo: "hexpm", optional: false]}], "hexpm"},
   "makeup_elixir": {:hex, :makeup_elixir, "0.10.0", "0f09c2ddf352887a956d84f8f7e702111122ca32fbbc84c2f0569b8b65cbf7fa", [:mix], [{:makeup, "~> 0.5.5", [hex: :makeup, repo: "hexpm", optional: false]}], "hexpm"},
   "nimble_parsec": {:hex, :nimble_parsec, "0.4.0", "ee261bb53214943679422be70f1658fff573c5d0b0a1ecd0f18738944f818efe", [:mix], [], "hexpm"},
-  "partisan": {:git, "https://github.com/lasp-lang/partisan.git", "d384d9cb31b6f82148db338e4bf72fc4b12999db", [branch: "rename-backend"]},
+  "partisan": {:git, "https://github.com/lasp-lang/partisan.git", "9c8e6d1dbf9c8bf2968df0cae69cce505e5cfe6e", [branch: "rename-backend"]},
```
Comment (cmeiklejohn) [inline mix.lock:10]: You cam bump to the latest version now.

File: lib/expartisan.ex [hunk 3]
```diff
@@ -82,6 +87,16 @@ defmodule ExPartisan do
...
   @doc """
   Disconnects the specified `node`.
+
+  ## Examples
+
+      iex(foo@127.0.0.1)> ExPartisan.members
+      {:ok, [:"foo@127.0.0.1", :"bar@127.0.0.1"]}
+      iex(foo@127.0.0.1)> ExPartisan.leave(bar)
+      :ok
+      iex(foo@127.0.0.1)> ExPartisan.members
```
Comment (cmeiklejohn) [inline lib/expartisan.ex:97]: Should we make members return a map?  That might be a good change across the board.

**Review by cmeiklejohn — APPROVED** [review]

File: lib/expartisan.ex (hunk 3 above)
Comment (jlerche) [reply lib/expartisan.ex:97]: Certainly wouldn't be a bad change, what kind of a shape were you thinking of?

File: lib/expartisan.ex (hunk 1 above)
Comment (jlerche) [reply lib/expartisan.ex:124]: The generated hexdocs will have a link in the typespec to its definition, I just pointed it out so people won't assume that's the case based on the given example.

File: lib/expartisan.ex (hunk 3 above)
Comment (cmeiklejohn) [reply lib/expartisan.ex:97]: After looking into this, this is a bigger change than initially thought.

**Pull request merged**